app/routes.py

```diff
@@ -21,11 +21,16 @@
 from bs4 import BeautifulSoup as bsoup
 from flask import jsonify, make_response, request, redirect, render_template, \
     send_file, session, url_for
-from requests import exceptions
+from requests import exceptions, get
 
 # Load DDG bang json files only on init
 bang_json = json.load(open(app.config['BANG_FILE']))
 
+# Get the newest version of WHOOGLE
+link_releases = "https://github.com/benbusby/whoogle-search/releases"
```

Comment:
App related constants like this should be in `app/__init__.py`, for example: `app.config['RELEASES_URL']` or something along those lines.